src/components/Discipline.js

```diff
@@ -57,17 +57,17 @@ const styles = EStyleSheet.create({
     fontFamily: 'OpenSans-Semibold',
     backgroundColor: '$transparent',
     color: '$textColor',
-    fontSize: 22,
-    lineHeight: 30,
-    width: '100% - 120'
+    fontSize: 28,
+    lineHeight: 38,
+    width: '100% - 140'
```

Comment:
I'm sure you've already checked this, but are you sure this calculation is working correctly? Just seems a bit odd to have `'100% - 140'`, semantically. Is this to adjust the size of items responsively depending on screen size?